### ISSUE TITLE
Configure prefixer via browserslist

### ISSUE THREAD

**Issue by lencioni** [Khan/aphrodite#239]
It looks like everybody is jumping on the browserslist train. I think it would be nice if we made it so the prefixes generated by Aphrodite could be configured in the same way that other tools like babel-preset-env and autoprefixer are configured.

More information: https://evilmartians.com/chronicles/autoprefixer-7-browserslist-2-released

**Comment by BabakMN** [comment]
I'd also like to be able to disable autoprefixing during development. It makes debugging easier (in the console).

**Comment by Kerumen** [comment]
I totally agree with @BabakMN. Currently, it's a pain to debug if you want to quickly disable / enable a CSS property as you have to do for all the prefixed-properties..

**Comment by cezaraugusto** [comment]
is there an intention to implement for this?
